docker/run-apache2.sh

```diff
@@ -3,6 +3,8 @@
 _DECK_HOST=$DECK_HOST
 _DECK_PORT=$DECK_PORT
 _API_HOST=$API_HOST
+_DECK_CERT=$DECK_CERT
```

Comment:
Ah darn, if the existing env vars depend on them being paths there's not much we can do.